kernel/src/utilities/mod.rs

```diff
@@ -4,6 +4,7 @@
 
 //! Utility functions and macros provided by the kernel crate.
 
+pub mod arch_helpers;
```

Comment:
Arguably should be a crate outside of kernel (e.g. `arch/common` or something?) but whatever for now...